meson.build

```diff
@@ -79,7 +79,7 @@ is_netbsd = host_machine.system() == 'netbsd'
 is_openbsd = host_machine.system() == 'openbsd'
 
 thread_dep = dependency('threads')
-fmt = dependency('fmt', version : ['>=5.3.0'], fallback : ['fmt', 'fmt_dep'])
+fmt = dependency('fmt', version : ['>=8.0.0'], fallback : ['fmt', 'fmt_dep'])
```

Comment:
This breaks compilation with subprojects as fmt 8.0.0 is not yet available from wrapdb.
Also, I believe it's possible to fix compilation in a way that does not require latest fmt on a rolling release.

Reply:
I agree, my bad!